build/azure-pipelines.yml

```diff
@@ -67,9 +67,15 @@ stages:
   - template: ../src/cli/.azuredevops/stage-publish-cli.yml
 
 #-endif
-- stage: Build_Staging
+- stage: Commit_Validation
 #-if false
   dependsOn: [] # This removes the implicit dependency on previous stage and causes this to run in parallel.
+#-endif
+  jobs:
+    - template: stage-commit-validation.yaml
+- stage: Build_Staging
+#-if false
```

Comment:
Why is there an `#-if false` here? The `Build_Staging` stage should always be dependeny on the Commit_Validation stage.